packages/manager/modules/pci/src/projects/project/private-networks/add/add.constants.js

```diff
@@ -13,9 +13,9 @@ export const VLAN_ID = {
 
 export const GUIDE_LINKS = {
   PRIVATE_NETWORK_WITH_GATEWAY:
-    'https://docs.ovh.com/gb/en/public-cloud/creating-private-network-with-gateway/',
+    'https://docs.ovh.com/fr/publiccloud/network-services/creating-private-network-with-gateway/',
```

Comment:
**question:** Are we sure to include only French pages? If i am not wrong, these docs are available in other languages as well.

Reply:
I think as usual we need based on subsidiary.

Reply:
as highlight by Guillaume, better to have a subs list.